protocol-units/bridge/shared/src/types.rs

```diff
@@ -219,6 +224,13 @@ impl From<Uint<256, 4>> for Amount {
 	}
 }
 
+//#[derive(Debug, PartialEq, Eq, Clone)]
+//enum State {
```

Comment:
Currently the enum series for the Ethereum side starts with 0.
There is also a big difference between how initiator and counterparty are calling states:

```
enum MessageState {
        INITIALIZED, vs PENDING
        COMPLETED,
        REFUNDED
    }
```
   
   On Movement they differ on the first and third state:
   
```
const INITIALIZED: u8 = 1; vs const LOCKED: u8 = 3;
    const COMPLETED: u8 = 2;
    const REFUNDED: u8 = 3; vs const CANCELLED: u8 = 3;
```
    
because 0 signifies uninitialized states on Ethereum, the Ethereum contract should be defined as 1,2,3 as not initialized states respectively as it could open up for unexpected attack vectors. We also should standardize nomeclature as well. Seems like simply using refunded makes more sense.
    
    This way, this shared enum could be used again in the /shared component:
    ```
    enum State {
    UNINITIALIZED,
    INITIALIZED,
    COMPLETED,
    REFUNDED
    }
    ```

Reply:
yeah this seems like a good solution. maybe good for a separate issue?

Reply:
Yes 0 would be `UNINITIALIZED`, on Eth, but by the time an event is emitted I don't think the state could ever possibly be 0 / unitialized.